EIPS/eip-6734.md

```diff
@@ -0,0 +1,531 @@
+---
+
+## Abstract
+
+The document describes a JSON token list that ensures that two or more Layer 1, Layer 2, or Sidechains can identify tokens from a different Layer 1, Layer 2, or Sidechain.
+
+## Motivation
+
+This particular work by the L2 WG of the EEA Communities Projects managed by OASIS, an open-source initiative, is motivated by a significant challenge around the definition and listing of tokens on Layer 1 (L1), Layer 2 (L2), and Sidechain systems. Note that for simplicity, this document we will collectively refer to L1, L2 and Sidechain systems as chains below since the challenge described below is valid across all such systems:
+
+* Consensus on the "canonical" token on chain B that corresponds to some token on chain A. When one wants to bridge token X from chain A to chain B, one must create some new representation of the token on chain B. It is worth noting that this problem is not limited to L2s -- every chain connected via bridges must deal with the same issue.
+
+Related to the above challenge is the standardization around lists of bridges and their routes across different chains. This will be addressed in a separate document. 
+
+Note that both of these issues are fundamental problems for the current multi-chain world.
+
+Therefore, the goal of this document is to help token users to operationalize and disambiguate the usage of a token in their systems.
+
+For lists of canonical tokens, L2s currently maintain their own customized versions of the Uniswap token list. For example, Arbitrum maintains a token list with various custom extensions. Optimism also maintains a custom token list, but with different extensions. It should be noted that both of these custom extensions refer to the bridge that these tokens can be carried through. However, these are not the only bridges that the tokens can be carried through, which means that bridges and token lists should be separated. Also note that currently, both Optimism and Arbitrum base "canonicity" on the token name + symbol pair.
+
+An example of an Arbitrum token entry is given below:
+
+```
+{
+logoURI: "https://assets.coingecko.com/coins/images/13469/thumb/1inch-token.png?1608803028",
+chainId: 42161,
+address: "0x6314C31A7a1652cE482cffe247E9CB7c3f4BB9aF",
+name: "1INCH Token",
+symbol: "1INCH",
+decimals: 18,
+extensions: {
+  bridgeInfo: {
+    1: {
+    tokenAddress: "0x111111111117dc0aa78b770fa6a738034120c302",
+    originBridgeAddress: "0x09e9222e96e7b4ae2a407b98d48e330053351eee",
+    destBridgeAddress: "0xa3A7B6F88361F48403514059F1F16C8E78d60EeC"
+     }
+   }
+  }
+}
+```
+
+This standard will build upon the current framework and augment it with concepts from the W3C DID Specification [[1]](https://www.w3.org/TR/2022/REC-did-core-20220719/) based on the JSON linked data model [[2]](https://www.w3.org/TR/2020/REC-json-ld11-20200716/) such as resolvable unique resource identifiers (URIs) and JSON-LD schemas which enable easier schema verification using existing tools.
```

Comment:
Please format all links as plain markdown links using the title of the standard as the anchor text.

```suggestion
This standard will build upon the current framework and augment it with concepts from [Decentralized Identifiers (DIDs)](https://www.w3.org/TR/2022/REC-did-core-20220719/) based on the JSON linked data model [JSON-LD](https://www.w3.org/TR/2020/REC-json-ld11-20200716/) such as resolvable unique resource identifiers (URIs) and JSON-LD schemas which enable easier schema verification using existing tools.
```